x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/slm/SnapshotRetentionConfigurationTests.java

```diff
@@ -103,13 +104,21 @@ public void testMaximum() {
     }
 
     public void testFailuresDeletedIfExpired() {
+        assertUnsuccessfulDeletedIfExpired(true);
+    }
+
+    public void testPartialsDeletedIfExpired() {
+        assertUnsuccessfulDeletedIfExpired(true);
```

Comment:
Should this be `false`? I think this is identical to the `testFailuresDeletedIfExpired` test above?

Reply:
Nice catch, thanks!